backend/t4cclient/extensions/modelsources/t4c/connection.py

```diff
@@ -81,14 +81,17 @@ def fetch_last_seen(mac_addr: str):
 
 
 def add_user_to_repository(
-    repository: str, username: str, password: str = generate_password()
+    repository: str,
+    username: str,
+    password: str = generate_password(),
+    isAdmin: bool = False,
 ):
     r = requests.post(
         config["modelsources"]["t4c"]["restAPI"] + "/users",
         params={"repositoryName": repository},
         json={
             "id": username,
-            "isAdmin": False,
+            "isAdmin": isAdmin,
```

Comment:
```suggestion
            "isAdmin": is_admin,
```

Please use snake_case for the python variables.